Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiTable.c

```diff
@@ -460,8 +460,8 @@ PlatformUpdateAcpiTable (
     //   1) will have a new Low Power S0 Idle Capable ACPI flag (bit offset 21).
     //
     if (FeaturesCfgData != NULL) {
-      if (FeaturesCfgData->Features.LowPowerIdle != 0 && PchIsSciSupported() != 1){
-        DEBUG ( (DEBUG_INFO, "Enabled Low Power S0 Idle Capable ACPI flag\n") );
+      if (FeaturesCfgData->Features.S0ix == 1){

```

Comment:
need considering DSO to update the ACPI table. if s0ix flag is added we could directly use that one.

Reply:
Done in UpdateS0ixStatus ()